docs/Documentation/model_introduction.md

```diff
@@ -2,11 +2,21 @@
 
 Every deep learning workflow begins with obtaining a model. You can choose to prepare a custom one, use a ready-made solution and adjust it to your needs, or even download and run a pre-trained network from an online database, such as OpenVINO's [Open Model Zoo](../model_zoo.md).  
 
-This section describes how to obtain and prepare your model for work with OpenVINO to get the best inference results:
-* [Browse a database of models for use in your projects](../model_zoo.md).
+[OpenVINO™ supports several model formats](../MO_DG/prepare_model/convert_model/supported_model_formats.md) and allows to convert them to it's own, OpenVINO IR, providing a tool dedicated to this task.
```

Comment:
I'm not sure what parts you think may not be necessary, so I may be stating the obvious here, but... When it goes to the "supported formats," yes, we need it. For two reasons. One is that the reader needs clear information on what the support looks like for different formats. For people who already know the product it seems obvious but documentation serves also, if not more, people new to the product. Secondly, we need node articles even if it means adding seemingly "unnecessary" content because grouping articles is crucial in making documentation clear. A user with some experience in OV should be able to navigate the entire thing just by article titles and nested sections. Also, we are preparing to change the menu, so that it presents the entire sitemap and has no headers for menu sections. That is why we currently have some "redundant" overview articles. They will become node articles for sections, hopefully.